topics/chip-seq/tutorials/formation_of_super-structures_on_xi/tutorial.md

```diff
@@ -0,0 +1,862 @@
+>    > >
+>    > >        ![Per sequence quality scores for read1](../../images/formation_of_super-structures_on_xi/read1_per_sequence_quality_scores.png "Per sequence quality scores")
+>    > >
+>    > >     - Homogeneous percentage of the bases
+>    > >
+>    > >        ![Per base sequence content for read1](../../images/formation_of_super-structures_on_xi/read1_per base_sequence_content.png "Per base sequence content")
+>    > >
+>    > >     - No N in the reads
+>    > >
+>    > >        ![Per base N content for read1](../../images/formation_of_super-structures_on_xi/read1_per_base_n_content.png "Per base N content")
+>    > >
+>    > >     - No duplicated sequences
+>    > >
+>    > >        ![Sequence Duplication Levels for read1](../../images/formation_of_super-structures_on_xi/read1_sequence_duplication_levels.png "Sequence Duplication Levels")
+>    > >
+>    > >     - No more known adapters
+>    > >
+>    > >        ![Adapter Content for read1](../../images/formation_of_super-structures_on_xi/read1_adapter_content.png "Adapter Content")
+>    > >  
+>    > > 2. The reads in `wt_H3K4me3_read1` are a bit worst:
+>    > >
+>    > >     - The "Per base sequence quality" is decreasing more at the end of the sequences, but it stays correct
+>    > >
+>    > >        ![Per base sequence quality for read2](../../images/formation_of_super-structures_on_xi/read1_per_base_sequence_quality.png "Per base sequence quality")
+>    > >
+>    > >     - The sequence `NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN` is over represented
+>    > >
+>    > >     Despite these small things, the overall quality of the reads is really good.
+>    > > 
+>    > > 3. If the quality of the reads is not good, we should:
+>    > >    1. Check what is wrong and think about it: it may come from the type of sequencing or what we sequenced (high quantity of overrepresented sequences in transcriptomics data, biaised percentage of bases in HiC data)
+>    > >    2. Ask the sequencing facility about it
+>    > >    3. Perform some quality treatment (in a reasonable way to not loose too much information) with some trimming or removal of bad reads
+>    > >
+>    > {: .solution }
+>    {: .question}
+>
+{: .hands_on}
+
+It is often necessary to trim sequenced read, for example, to get rid of bases that were sequenced with high uncertainty (= low quality bases).
+
+> ### {% icon question %} Questions
+>
+> What is the relation between `wt_H3K4me3_read1` and `wt_H3K4me3_read2`?
+>
+> > ### {% icon solution %} Solution
+> > The ChIP-seq data for `wt_H3K4me3_rep1` has been sequenced using paired-end sequencing. 
+> >
+> > The paired-end sequencing is based on the idea that the initial DNA fragments (longer than the actual read length) is sequenced on both its ends. This approach results in two reads per fragment, with the first read in forward orientation and the second read in reverse-complement orientation.
```

Comment:
s/ is sequenced on both its ends/ is sequenced from both sites/
Add: The distance between both reads is known and therefor an additional information that can improve read mapping.